cobigen-cli/cli/src/main/assembly/resources/bin/cg

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 LIB_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../lib" && pwd )"
 if [ -z "$1" ]; then
```

Comment:
This looks better, now. Can you please add a comment in the file that -cp "$LIB_DIR/*" is not working? Just to make sure that it is not changed back and everyone knows directly why it is done the way it is done.

Reply:
Comment was added.